Mathlib/Geometry/Manifold/ContMDiff/Basic.lean

```diff
@@ -162,11 +162,20 @@ end id
 
 section Iterate
 
-/-- Iterated `ContMDiff` functions are `ContMDiff`. -/
-theorem ContMDiff.iterate {f : M → M} (fa : ContMDiff I I n f) (k : ℕ) :
+/-- The iterates of `C^n` functions are `C^n`. -/
+theorem ContMDiff.iterate {f : M → M} (hf : ContMDiff I I n f) (k : ℕ) :
```

Comment:
Can you golf the proof of this result using `ContMDiffOn.iterate`? (For instance, `contMDiffOn_univ` should be useful.)